internal/service/teams/resource_teams_test.go

```diff
@@ -105,15 +102,15 @@ func TestAccConfigRSTeam_importBasic(t *testing.T) {
 			},
 			{
 				ResourceName:      resourceName,
-				ImportStateIdFunc: testAccCheckMongoDBAtlasTeamStateIDFunc(resourceName),
+				ImportStateIdFunc: importStateIDFunc(resourceName),
 				ImportState:       true,
 				ImportStateVerify: true,
 			},
 		},
 	})
 }
 
-func testAccCheckMongoDBAtlasTeamExists(resourceName string, team *matlas.Team) resource.TestCheckFunc {
+func checkExists(resourceName string, team *admin.Team) resource.TestCheckFunc {
```

Comment:
i don't think team param is needed, i've changed in other resources to have checkExists only with resourceName and use normal resource.TestCheckResourceAttr for checking this